gn3/computations/correlations.py

```diff
@@ -99,8 +100,13 @@ def do_bicor(x_val, y_val) -> Tuple[float, float]:
 package :not packaged in guix
 
     """
-    x_val, y_val = 0, 0
-    return (x_val, y_val)
+
+    try:
+        results = call_biweight_script(x_val, y_val)
+    except Exception as error:
+        raise error
+
+    return results
```

Comment:
**suggestion:**
```suggestion
    try:
        results = call_biweight_script(x_val, y_val)
        return results
    except Exception as error:
        raise error
```

I normally prefer having variables in the same block scope.